tests/filecheck/dialects/arith/arith_ops.mlir

```diff
@@ -10,142 +10,215 @@
 
```

Comment:
Note, we don't have to test both the generic and the non-generic version here.
The generic version will always round-trip, because that's the parser role to make it round-trip.
The custom version might not, which is why we check it.
At the same time, printing the generic version allows us to check that it correspond to what we expect from the custom version.
So I would just drop the generic input here.